src/sentry/models/release.py

```diff
@@ -74,6 +98,11 @@ class Release(Model):
     projects = models.ManyToManyField(
         "sentry.Project", related_name="releases", through=ReleaseProject
     )
+    status = BoundedPositiveIntegerField(
+        default=0,
```

Comment:
Typically you don't want to set a default like this. The problem is that Django will create the default in postgres and drop it.
```
ALTER TABLE "sentry_release" ADD COLUMN "status" integer DEFAULT 0 NOT NULL CHECK ("status" >= 0);
ALTER TABLE "sentry_release" ALTER COLUMN "status" DROP DEFAULT;
```
Adding a default in postgres means rewriting the table, so we'd end up taking sentry down if we did it. What you probably want to do here is use `SeparateDatabaseAndState`, and just have the default set in Django but not in the database.

So your state operation would look like
```
        migrations.AddField(
            model_name='release',
            name='status',
            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(choices=[(0, 'Open'), (1, 'Archived')], default=0),
        ),
 ```
and your database operation would look like
```
        migrations.AddField(
            model_name='release',
            name='status',
            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(choices=[(0, 'Open'), (1, 'Archived')]),
        ),
 ```

Reply:
Thanks, I have changed this to be raw SQL now because I see no precedent for having AddField in a database operation. I have also reverted all the model changes as they are not strictly necessary to deploy this change. In the next PR we can then create the model and start using the column.